package.json

```diff
@@ -11,12 +11,12 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@creit.tech/stellar-wallets-kit": "1.3.0",
+    "@creit.tech/stellar-wallets-kit": "1.7.3",
     "@headlessui/react": "^1.7.18",
     "@script3/soroban-governor-sdk": "^1.3.2",
     "@snapshot-labs/pineapple": "^1.1.0",
-    "@stellar/freighter-api": "1.7.0",
-    "@stellar/stellar-sdk": "13.1.0",
+    "@stellar/freighter-api": "4.1.0",
```

Comment:
is there a reason this is included separately? We probs don't use it directly anymore